### ISSUE TITLE
Misc narrow/responsive issues throughout settings/admin/mod area

### ISSUE THREAD

**Issue by mjankowski** [mastodon/mastodon#32743]
### Steps to reproduce the problem

Small sample...

Dates should probably wrap?...

![Image](https://github.com/user-attachments/assets/0306f430-bcf2-4a57-b3f7-cccf7b0d03ff)

Full browser description should maybe truncate?..

![Image](https://github.com/user-attachments/assets/2ef31587-48ae-48ab-9b85-c97fdd4d5703)

Recommended badge should bump over/under text?...

![Image](https://github.com/user-attachments/assets/af661608-ac26-41d0-8cf6-2ce87964e761)

Struggling table...

![Image](https://github.com/user-attachments/assets/24a9636d-571c-425c-9d43-61b73f31b9c0)

### Expected behaviour

I should be delighted

### Actual behaviour

I'm concerned

### Detailed description

Many of the "newer" views here are reasonable responsively, but some of the older table-based ones struggle a little, and would benefit from a once over, maybe use flexbox or grid approach and lose some tables?

Want to make sure there's at least vague agreement on this concern before doing anything. Is there already any WIP for this area? Are there any areas which have a solid approach we'd like to replicate and could borrow from?

### Mastodon instance

localhost:3000

### Mastodon version

v4.4.0

### Browser name and version

Safari

### Operating system

mac

**Comment by ThisIsMissEm** [comment]
Yeah, pretty much all the "tables" in the moderation/admin UI are messed up like this on small screens. There is another issue open that attempts to tackle this. https://github.com/mastodon/mastodon/issues/32376
